packages/web/src/pages/sign-in-page/SignInPage.tsx

```diff
@@ -39,21 +52,72 @@ export const SignInPage = () => {
   )
 
   return (
-    <div>
-      <h1>{messages.header}</h1>
-      <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-        <Form>
-          <TextField name='email' label={messages.emailLabel} />
-          <TextField name='password' label={messages.passwordLabel} />
-          <Button text={messages.signIn} type='submit' />
-        </Form>
-      </Formik>
-      <ButtonTmp
-        // @ts-ignore
-        as={Link}
-        to={SIGN_UP_PAGE}
-        text={messages.createAccount}
-      />
-    </div>
+    <Flex h='100%' alignItems='center' justifyContent='center'>
+      <PageWithAudiusValues>
+        <Flex
+          className={styles.root}
+          h='100%'
+          w={480}
+          // want 80px but don't have var for it
+          pv='4xl'
+          ph='2xl'
+          direction='column'
+          gap='2xl'
+          justifyContent='space-between'
+        >
+          {/* TODO: confirm 40px spacing value */}
+          <Flex direction='column' gap='2xl' alignItems='center'>
+            <PreloadImage
+              src={audiusLogoColored}
+              className={styles.logo}
+              alt='Audius Colored Logo'
+            />
+            <Flex w='100%' direction='row' justifyContent='flex-start'>
+              <Text variant='heading' size='l' tag='h1' color='heading'>
+                {messages.title}
+              </Text>
+            </Flex>
+            <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+              <Box w='100%'>
+                <Form>
+                  <Flex direction='column' gap='2xl' w='100%'>
+                    <Flex direction='column' gap='l'>
+                      <TextInput name='email' label={messages.emailLabel} />
+                      {/* TODO: password visibility icon and toggle */}
+                      <TextInput
+                        name='password'
+                        label={messages.passwordLabel}
+                        endIcon={IconVisibilityHidden}
+                        type='password'
+                      />
+                    </Flex>
+                    <Flex direction='column' gap='l'>
+                      <Button
+                        text={messages.signIn}
+                        iconRight={IconArrowRight}
+                        type='submit'
+                      />
+                      <Flex direction='row' alignItems='flexStart'>
+                        <Text color='heading'>
+                          {/* TODO: link destination */}
+                          <Link to={''}>{messages.forgotPassword}</Link>
+                        </Text>
+                      </Flex>
+                    </Flex>
+                  </Flex>
+                </Form>
+              </Box>
+            </Formik>
+          </Flex>
+          <Button
+            // @ts-ignore
+            asChild={Link}
```

Comment:
link doesn't seem to work this way

Reply:
yeah it only works with old stems button! will add a pr for the new button